
string service_sequence
---
bool received